### ISSUE TITLE
[BUG]: Keyboard Emulation does not allow for EVA UP (Left Shift)

### ISSUE THREAD

**Issue by CodapopKSP** [LRTNZ/KerbalSimpitRevamped#61]
Keyboard Emulation presses Right Shift, but Left Shift is needed for EVA UP using the jet pack. No other function in Simpit allows for EVA Up.

Mod version: Latest Pre-Release
Game version: 1.11
Operating system: Win10
Microprocessor/s in use: Arduino Nano, Intel 8700K

This is similar to the bug regarding sending Shift as a key modifier, but in this case it does not send as a normal keypress, either. This bug could be remedied by remapping keys in KSP (however, the other bug cannot), but finding a solution to one of these bugs is likely to also fix the other.

**Comment by PBechon** [comment]
Bug probably fixed in 8fbbd47542ff84ab32f39737cbad3b14f990bae5

Here is a code that is working on my side to go up while in EVA during 1 sec

```
  keyboardEmulatorMessage msg(0xA0, KEY_DOWN_MOD);
  mySimpit.send(KEYBOARD_EMULATOR, msg); 

  delay(1000);

  msg.modifier = 16;
  mySimpit.send(KEYBOARD_EMULATOR, msg);
```

Replace `A0` with `A2` for doing down (LCONTROL instead of LSHIFT). All those values can be found in KerbalSimpit\Contrib\InputSimulator\WindowsInput\Native\VirtualKeyCode.cs

Can you check on your side ?

**Comment by PBechon** [comment]
Resolution confirmed, and an example was added to show a nominal way of controlling EVA up
